### PR TITLE
Proposal: Developing a CS Teaching Methods Experiment

### PR DESCRIPTION
**Pull request by PocketOfWeird**

### CODE REVIEW

File: _posts/2019-12-03-proposal-cs-teaching-methods-experiment.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Proposal: Developing a CS Teaching Methods Experiment"
+tags: ["csc696", "teaching", "pedagogy", "experiment preparation", "msu"]
+author: "Nathan Hartzler"
+---
+
+# Overview
+
+Pedagogy for Computer Science is still a young area of research where many contributions can be made. I would like to conduct research on a multi-sectioned Missouri State Univeristy Department of Computer Science course in the Fall 2020 semester. I will use this proposed Spring 2020 CSC696 course to develop a hypothesis and build an expirment for the Fall 2020 semester.
```
Comment (anthonyjclark) [inline _posts/2019-12-03-proposal-cs-teaching-methods-experiment.md]:
```suggestion
Pedagogy for Computer Science is still a young area of research where many contributions can be made. I would like to conduct research on a multi-sectioned Missouri State University Department of Computer Science course in the Fall 2020 semester. I will use this proposed Spring 2020 CSC696 course to develop a hypothesis and build an experiment for the Fall 2020 semester.
```

File: _posts/2019-12-03-proposal-cs-teaching-methods-experiment.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Project Milestones
+
+* Develop a good hypothesis
+    * Does a tactile demonstration for a theoretical computer science concept increase student understanding of the concept?
```
Comment (anthonyjclark) [inline _posts/2019-12-03-proposal-cs-teaching-methods-experiment.md]: Elaborate on what you mean by tactile demonstration. Maybe given an example.

File: _posts/2019-12-03-proposal-cs-teaching-methods-experiment.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    * Test all 4 sections on the same set of Turing Machine questions
+        * Use as a set of bonus questions on one of the course exams
+* By February 24th, 2020 
+    * Submit an application for review to Missouri State University's [Institutional Review Board](https://ora.missouristate.edu/IRB.htm)
+* By April 6th, 2020
+    * Present the proposed research to the instructors of a multi-sectioned course
+
+## Meetings and Work
+
+Meetings would be held about once a week for the duration of the Spring 2020 semester. In addition to working towards the above project milestones, I would also use the semester to read STEM Pedagogy methods and Computer Science Pedagogy research papers. The hope would be that this research can be repeated in future semesters.
```
Comment (anthonyjclark) [inline _posts/2019-12-03-proposal-cs-teaching-methods-experiment.md]:
```suggestion
Meetings would be held about once a week for the duration of the Spring 2020 semester. Besides working towards the above project milestones, I would also use the semester to read STEM Pedagogy methods and Computer Science Pedagogy research papers. The hope would be that I can repeat this research in future semesters.
```

**Comment by PocketOfWeird** [comment]
The requested changes have been made.

**Pull request merged**